cfme/tests/distributed/test_appliance_replication_manual.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+# pylint: skip-file
+"""Manual tests"""
+import pytest
+
+from cfme import test_requirements
+
+pytestmark = [pytest.mark.ignore_stream('5.10', 'upstream')]
```

Comment:
Throughout these modules, you can include `pytest.mark.manual` here at the module level, so its not repeated fro every test function. You can also move requirement markers here too if they're all the same.